src/main/java/io/jenkins/plugins/jfrog/JfStep.java

```diff
@@ -119,14 +119,14 @@ private boolean shouldConfig(FilePath jfrogHomeTempDir) throws IOException, Inte
     /**
      * Locally configure all servers that was configured in the Jenkins UI.
      */
-    private void configAllServers(Launcher.ProcStarter launcher, String jfrogBinaryPath, boolean isWindows) throws IOException, InterruptedException {
+    private void configAllServers(Launcher.ProcStarter launcher, String jfrogBinaryPath, boolean isWindows, Job<?, ?> parent) throws IOException, InterruptedException {
```

Comment:
The "parent" of the Run object is the Job configuration. I suggest to rename it to job:
```suggestion
    private void configAllServers(Launcher.ProcStarter launcher, String jfrogBinaryPath, boolean isWindows, Job<?, ?> job) throws IOException, InterruptedException {
```